compiler-rt/lib/builtins/fp_extend.h

```diff
@@ -81,6 +81,13 @@ static inline int src_rep_t_clz_impl(src_rep_t a) {
 
 #define src_rep_t_clz src_rep_t_clz_impl
 
+#elif defined SRC_BFLOAT
```

Comment:
The changes in `compiler-rt` were from https://reviews.llvm.org/D151436.

Reply:
I doubt whether the implementation handles exception or not. But I don't have any suggestion to it.